Update PULL_REQUEST_TEMPLATE.md (#1687)
FreeTubeApp/FreeTube: .github/PULL_REQUEST_TEMPLATE.md

Title
---

> **Important notes**
> Please note that only PrestoN is able to merge Pull Requests into master.

Reviewer: ```suggestion
**Important note**
```
We should probably remove this line at this point given that it is both inaccurate and does not provide salient information.

> Please note that if u incorrectcly filled out or not filled out the PR template, it will result in the closure of this PR.

Reviewer: ```suggestion
We may remove your pull request if you do not use this provided PR template correctly.
```
Spelling errors, wording adjustment.

**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [ ] Feature Implementation

**Related issue**

> Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #99999". If this pull request fully resolves multiple relevant issues, put "closes" before both issue numbers. Example: "closes #99999, closes #12345".

Reviewer: ```suggestion
Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #123456".
```

I think your second added sentence is not necessary given that this has only rarely been a problem.

Author: Yeah that makes sense. I guess i just wanted to be clear about the way it was done.
